src/Mod/TechDraw/TechDrawTools/CommandHoleShaftFit.py

```diff
@@ -68,5 +64,4 @@ def IsActive(self):
 
 #
 # The command must be "registered" with a unique name by calling its class.
-Gui.addCommand('TechDraw_HoleShaftFit', CommandHoleShaftFit())
-
+Gui.addCommand('TechDraw_HoleShaftFit', CommandHoleShaftFit())
```

Comment:
> Final newline missing (missing-final-newline)

Add final newline

Reply:
I know i asked you to remove this in the first place but I didn't realize it was at the end of the file